packages/new-auth-ui/src/core/private-route.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useEffect, LazyExoticComponent, FC, useState, Dispatch, SetStateAction } from 'react'
+import { History } from 'history'
+import { Route, RouteProps, useHistory } from 'react-router'
+import { useReapitConnect, ReapitConnectSession } from '@reapit/connect-session-next'
+import { reapitConnectBrowserSession } from '../core/connect-session'
+import { useGlobalState } from './use-global-state'
+import { SendFunction, useReapitUpdate } from '@reapit/utils-react'
+import { MemberModel, UpdateMemberModel } from '@reapit/foundations-ts-definitions'
+import { DATE_TIME_FORMAT, UpdateActionNames, updateActions } from '@reapit/utils-common'
+import dayjs from 'dayjs'
+
+export interface PrivateRouteProps {
+  component: FC | LazyExoticComponent<any>
+  exact?: boolean
+}
+
+export const handleRedirectRegistraitionPage =
```

Comment:
all this members stuff can go I think

Reply:
TBH, I think you can just use a vanilla Route in your Router and get rid of PrivateRoute